app/views/rooms/index.html.haml

```diff
@@ -0,0 +1,13 @@
+.container
+  %h2 ルーム一覧
+  - @rooms.each do |room|
+    .room-content
+      = link_to room do
+        = room.name
+      = "#{room.reviewees.count} / #{room.capacity}"
```

Comment:
__[HAML-Lint]__

`= "..."` should be rewritten as `...`


<!-- Comment by [SideCI](https://sideci.com) -->